src/main/java/net/syscon/elite/service/CourtHearingCancellationService.java

```diff
@@ -23,6 +24,7 @@ public CourtHearingCancellationService(final CourtEventRepository courtEventRepo
     @Transactional
     @VerifyBookingAccess
     @HasWriteScope
+    @PreAuthorize("hasRole('PECS_PRISON')")
```

Comment:
Very minor should this be something like 'COURT_HEARING_MAINTAINER' which is assigned to the PECS system credentials

Reply:
Hmmm not sure as this what came about talking with @Mjwillis

Reply:
Updated.